### PR TITLE
updating the latest versions for c7 image 

### PR DESCRIPTION
**Pull request by bindushree16**
https://github.com/Shippable/pm/issues/11515

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -84,16 +84,25 @@ echo "================= Installing Java  ==================="
...
-sudo yum install http://opensource.wandisco.com/centos/7/git/x86_64/wandisco-git-release-7-2.noarch.rpm
-sudo yum install git-2.18.0
+sudo yum install curl-devel expat-devel gettext-devel openssl-devel zlib-devel
+sudo yum install gcc perl-ExtUtils-MakeMaker
+cd /usr/src
+wget https://www.kernel.org/pub/software/scm/git/git-"$GIT_VERSION".tar.gz
+tar xzf git-"$GIT_VERSION".tar.gz
+cd git-"$GIT_VERSION"
+make prefix=/usr/local/git all
+make prefix=/usr/local/git install
```
Comment (rageshkrishna) [inline install.sh]: @ric03uec Should we really compile, or just not update it until it shows up in a package repo?

Comment (ric03uec) [reply install.sh]: ya. lets just wait for the official package to get updated for this. no need to build from source

File: node/install.sh [hunk 2]
```diff
@@ -8,17 +8,17 @@ curl https://raw.githubusercontent.com/creationix/nvm/$NVM_VERSION/install.sh |
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 
-export NODEJS_VERSION=8.12*
+export NODEJS_VERSION=10.13*
```
Comment (rageshkrishna) [inline node/install.sh]: Where is this being used?

File: node/install.sh [hunk 3]
```diff
@@ -8,17 +8,17 @@ curl https://raw.githubusercontent.com/creationix/nvm/$NVM_VERSION/install.sh |
...
 echo "================= Installing nodejs $NODEJS_VERSION ==================="
-curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
+curl --silent --location https://rpm.nodesource.com/setup_10.x | sudo bash -
 #adding key required to install nodejs
 rpm --import /etc/pki/rpm-gpg/NODESOURCE-GPG-SIGNING-KEY-EL
 sudo yum install nodejs
 npm install npm@"$NPM_VERSION" -g
 
-export YARN_VERSION=1.10*
+export YARN_VERSION=1.12*
```
Comment (rageshkrishna) [inline node/install.sh]: Where is this being used?

**Pull request merged**